src/extension.ts

```diff
@@ -184,8 +183,8 @@ export async function activate(context: vscode.ExtensionContext) {
     const completionProvider = new CompletionProvider();
     context.subscriptions.push(vscode.languages.registerCompletionItemProvider(ARDUINO_MODE, completionProvider, "<", '"', "."));
 
-    usbDetector = new UsbDetector(context.extensionPath);
-    usbDetector.startListening();
+    UsbDetector.extensionRoot = context.extensionPath;
```

Comment:
You can add Initialize(extensionRoot : string) method to UsbDector instance for passing value but not use the module to pass object value